app/helpers/container_helper/textual_summary.rb

```diff
@@ -32,4 +32,8 @@ def textual_restart_count
   def textual_backing_ref
     {:label => "Backing Ref (Container ID)", :value => @record.backing_ref}
   end
+
+  def textual_command
+    {:label => "Command", :value => @record.container_definition.command  } if @record.container_definition.command
```

Comment:
@matthewd yeah sounds good to me. Let's favor simplicity vs structured data (hbase) for now.